<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_load">
         <property name="text">
          <string>load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="ImageView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="topMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Affine</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Affined">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <item>
            <property name="text">
             <string>Right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Left</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Up</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Down</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Front</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Back</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="AffinedImageView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Threshold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ColorThreshed">
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yellow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>White</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Orange</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="ColorThreshedImageView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>H_HIGH:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_h_high"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>H_LOW:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_h_low"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>S_HIGH:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_s_high"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>S_LOW:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_s_low"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>V_HIGH:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_v_high"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>V_LOW:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_v_low"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_changeHSV">
         <property name="text">
          <string>send</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Point</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="pointInfo"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_changeP">
         <property name="text">
          <string>send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
